packages/components/src/accordion/src/Accordion.tsx

```diff
@@ -160,6 +160,12 @@ export function InnerAccordion({
 
 InnerAccordion.defaultElement = DefaultElement;
 
+/**
+ * Accordions are commonly used to reduce the need to scroll when presenting multiple sections of content on a single page.
+ *
+ * [Documentation](https://orbit.sharegate.design/?path=/docs/accordion--default-story)
+*/
```

Comment:
ideally the comments would be directly next to the component without the extra line. But it still works